addons/survey/models/survey_survey.py

```diff
@@ -317,6 +297,17 @@ def _compute_session_question_answer_count(self):
             )[0]
             survey.session_question_answer_count = answer_count
 
+    @api.depends('access_token')
```

Comment:
I tried what Nihar did and indeed it did not work with access_token but it did with title. If that's incorrect/incomplete, you can safely ignore my comment :)

Reply:
Indeed, it did not, but that was because access_token was not in the form, which was my mistake :slightly_smiling_face: 

I added it as invisible and it seems to be working fine now.

Reply:
Ah, that's why then. Thanks!